conformance/base/manifests.yaml

```diff
@@ -413,3 +413,49 @@ spec:
         resources:
           requests:
             cpu: 10m
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mirrored-backend
```

Comment:
Why do we need a new Service and Deployment for this test? Could we reuse one of the existing ones?

Reply:
I initially reused one and changed it based on your comment here https://github.com/kubernetes-sigs/gateway-api/pull/1912#discussion_r1158957007

Would you prefer that I change one of the existing backends to have only 1 replica instead of having a new one? 

Anyway I cover cases where there is more than one pod (I read the logs from all the pods and break the loop once I found the line I want) so in theory I could remove this new one and not doing any change to the base manifests.

Reply:
>Anyway I cover cases where there is more than one pod (I read the logs from all the pods and break the loop once I found the line I want) so in theory I could remove this new one and not doing any change to the base manifests.

Since you're already doing this, maybe just use an existing Service/Deployment.